P14-C02-multiple-gpus-scratch.ipynb

```diff
@@ -61,11 +61,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Using all of GPUs together often significantly speedup the training. Different to the CPU that all CPU cores are used in default, however, most layers can only run on a single GPU. Therefore, additional works are required to partition workloads into multiple GPUs. \n",
+    "Using all of the GPUs together often significantly speeds up the training. As compared to CPU where all the CPU cores are used by default, most layers of a neural network can only run on a single GPU. Therefore, additional workers are required to partition the workload into multiple GPUs. \n",
     "\n",
     "## Data Parallelism\n",
     "\n",
-    "Data parallelism is the widely used approach to partition workloads. Assume there are *k* GPUs, it split the examples in a data batch into *k* parts, each GPU will compute the gradient on a part of the batch, gradients are then summed across all GPUs before updating the weights.\n",
+    "Data parallelism is a widely used approach to partition workloads. Assume there are *k* GPUs, it splits the examples in a data batch into *k* parts, such that each GPU will compute the gradient on a part of the batch; gradients are then summed across all GPUs before updating the weights.\n",
```

Comment:
Please ditch the semicolong. Make it a full-stop and then a new sentence.

"a part of the batch; gradients are then"
-> "a part of the batch. Gradients are then"